package.json

```diff
@@ -33,7 +33,7 @@
   "license": "MIT",
   "dependencies": {
     "@wikia/ad-engine": "git+https://github.com/Wikia/ad-engine.git#v13.1.2",
-    "@wikia/ad-products": "git+https://github.com/Wikia/ad-products.git#v8.5.1",
+    "@wikia/ad-products": "git+https://github.com/Wikia/ad-products.git#v9.0.0",
```

Comment:
FYI:
in mobile-wiki we have renovate setup
if you'd release these packages to npm - we could have it here updated automatically